policy/pa.go

```diff
@@ -74,22 +74,17 @@ type blockedNamesPolicy struct {
 	AdminBlockedNames []string `yaml:"AdminBlockedNames"`
 }
 
-// SetHostnamePolicyFile will load the given policy file, returning error if it
-// fails. It will also start a reloader in case the file changes
+// SetHostnamePolicyFile will load the given policy file, returning an error if
+// it fails.
 func (pa *AuthorityImpl) SetHostnamePolicyFile(f string) error {
```

Comment:
nit: I would consider renaming this function, because now it's more like "load a hostname policy file", rather than "set the file to load from, and start a reloader to handle it". Maybe `LoadHostnamePolicyFile`?

But that does mean touching a lot more files, so maybe not worth it.

Reply:
I'll make the change, sounds good to me.